lib/ansible/utils/module_docs_fragments/oneview.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+class ModuleDocFragment(object):
+
+    # OneView doc fragment
+    DOCUMENTATION = '''
+options:
+    config:
+      description:
+        - Path to a .json configuration file containing the OneView client configuration.
+          The configuration file is optional. If the file path is not provided, the configuration will be loaded from
+          environment variables.
```

Comment:
It could be useful to specify that the config file is on the managed machine rather than the controller.  This bites some people sometimes.  Should probably also mention what the environment variables the configuration is loaded from are.